build/build-treemap.js

```diff
@@ -8,6 +8,21 @@
 const fs = require('fs');
 const GhPagesApp = require('./gh-pages-app.js');
 
+function buildLocales() {
+  const locales = require('../lighthouse-core/lib/i18n/locales.js');
+  const clonedLocales = JSON.parse(JSON.stringify(locales));
+
+  for (const lhlMessages of Object.values(clonedLocales)) {
+    for (const key of Object.keys(lhlMessages)) {
+      if (!key.startsWith('lighthouse-treemap')) {
+        delete lhlMessages[key];
+      }
+    }
+  }
+
+  return 'const locales =' + JSON.stringify(clonedLocales, null, 2) + ';';
```

Comment:
with something this specially injected, WDYT about the name being a little more in your face? 

`BUILD_INJECTED_LOCALES`/`__buildInjectedLocales__`/`sOmEtHiNgElSe`?

Reply:
I don't think the cloak and dagger is necessary here, since we control the code that runs here (and it's not a library) :)

Reply:
self-describing variable names >> comments :)

but yes a comment would somewhat address my concern too.